stylelint.config.js

```diff
@@ -235,5 +238,17 @@ module.exports = {
 				],
 			}
 		},
+		{
+			/* Module CSS file classes have an underscore before classes, and use underscores instead of a dash as separators (e.g. _spectrum_well) */
+			files: ["tools/bundle/**/*.module.css"],
```

Comment:
Do you know if this works if we simplify it to just `files:["*.module.css"],` - I'm just thinking all module files would have these same linting issues so if we add more, it'd be lovely to be able to inherit these automatically

Reply:
That should work. I can test and make that change.